lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -944,6 +961,18 @@ export class UserAgentApplication {
       // resolve/reject based on cacheResult
       if (cacheResultResponse) {
         this.logger.info("Token is already in cache for scope:" + scope);
+
+        if (cacheResultResponse.accessTokenType === AuthenticationType.PoP) {
+            const signedPopToken = await this.popTokenGenerator.signToken(cacheResultResponse.accessToken);
+
+            resolve({
+                ...cacheResultResponse,
+                accessToken: signedPopToken
+            });
+
+            return null;
```

Comment:
either 

return; or return resolve({ ... 
returning null seems to say empty value instead of what I think is meant here which is void return

Reply:
Yeah, think I was just trying to be consistent with teh code around it.